project/plugins.sbt

```diff
@@ -1,18 +1,14 @@
-import sbt._
-
 resolvers += Resolver.url( "HMRC Sbt Plugin Releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))( Resolver.ivyStylePatterns)
-resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "0.8.5")
+resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
```

Comment:
you can update all the resolvers to use Resolver functions, e.g.:
```
resolvers += Resolver.bintrayIvyRepo("hmrc", "sbt-plugin-releases")
resolvers += Resolver.typesafeRepo("releases")
```

Reply:
Have just pushed another commit to do this.